From d2e20fc3fba1eb540fa80a116ffd662c5b36ed07 Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@nxp.com>
Date: Thu, 14 Dec 2017 15:25:26 +0800
Subject: [PATCH 3105/5242] MLK-17215-1 video: fbdev: mipi_dsi_northwest: fix
 an integer overflow issue

commit  91652a431f5c64c9446b8609f3a88fb1f7a70b0c from
https://source.codeaurora.org/external/imx/linux-imx.git

On ARM32 socs, the 'UL' and 'ULL' postfix are different.
And if using a 64bit constant integer, 'ULL' is the right
postfix.

Signed-off-by: Fancy Fang <chen.fang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/video/fbdev/mxc/mipi_dsi_northwest.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/video/fbdev/mxc/mipi_dsi_northwest.c b/drivers/video/fbdev/mxc/mipi_dsi_northwest.c
index 9635785..c26578e 100644
--- a/drivers/video/fbdev/mxc/mipi_dsi_northwest.c
+++ b/drivers/video/fbdev/mxc/mipi_dsi_northwest.c
@@ -53,7 +53,7 @@
 #define NS2PS_RATIO			(1000)
 #define	MIPI_LCD_SLEEP_MODE_DELAY	(120)
 #define MIPI_FIFO_TIMEOUT		msecs_to_jiffies(250)
-#define PICOS_PER_SEC			(1000000000UL)
+#define PICOS_PER_SEC			(1000000000ULL)
 #define PICOS2KHZ2(a, bpp)		\
 	DIV_ROUND_CLOSEST_ULL(PICOS_PER_SEC * (bpp), (a))
 
-- 
1.7.9.5

